### DISCUSSION TITLE
Why Chrome?

### DISCUSSION THREAD

**Discussion by Anutrix** [FlareSolverr/FlareSolverr#749] [Q&A]
Why was the switch to Chrome/Chromium made?

I couldn't find any MR/issue regarding the reason behind the switch. Do share

**Comment by ngosang** [comment]
Puppeter has poor support for Firefox and the FlareSolverr code was difficult to maintain. At a certain point Firefox started to be detected by Cloudflare and it was not possible to skip the challenge. FlareSolverr v3 uses Selenium + Chrome + Python.
You can see the old code here => https://github.com/FlareSolverr/FlareSolverr2
Feel free to fork and play with it. I'm not going to maintain it anymore, but maybe someone else can make it work again.
